addusers.py

```diff
@@ -46,25 +46,8 @@
 from quotas import QuotaManager
 from setpass import SetpassClient, random_password
 from config import set_config_file
-
-
-class InvalidEmailError(Exception):
-    """User's email address does not pass basic format validation"""
-
-
-class ItemExistsError(Exception):
-    """Keystone resource already exists and cannot be created"""
-    def __init__(self, item_type, item_name):
-        msg = "{} exists with the name: {}".format(item_type, item_name)
-        super(ItemExistsError, self).__init__(msg)
-
-
-class ItemNotFoundError(Exception):
-    """The specified Keystone resource was not found"""
-    def __init__(self, item_type, item_name):
-        msg = "No {} found in Keystone with name: {}".format(item_type,
-                                                             item_name)
-        super(ItemNotFoundError, self).__init__(msg)
+from moc_exceptions import InvalidEmailError, ItemExistsError
+from moc_exceptions import ItemNotFoundError
```

Comment:
I'm guessing there are 2 lines here because otherwise the line is too long? 

Python allows you to do multiline imports like this:
```
from foo import (classA, classB, classC,
                 classD)
```
What you have is not wrong and works, but grouping them with `()` this way is cleaner.  It makes it easier for other people to see easily what is imported form moc_exceptions.  Otherwise it's easy to not notice the additional line(s).